src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
@KaiHa Ah, sorry for the confusion. I did check recently that Hadrian doesn't build these packages by default, but I didn't realise that Hadrian still parsed their `cabal` files in order to determine overall package dependencies!

I've just pushed a fix for this, so I think this should no longer be a problem for you -- the build should succeed on the extracted sources.